### ISSUE TITLE
Inconsistent Behavior Between Match and Swap

### ISSUE THREAD

**Issue by shacthulu** [spencermountain/compromise#1011]
Hey, FANTASTIC library.  I'm finding a slight inconsistency in the documentation.  .swap is documented as using traditional match syntax.  However, I'm finding that to not be the case when it comes to possessives.
Reproduced in the compromise-root page (https://observablehq.com/@spencermountain/compromise-root)

```
{
  let doc = nlp('Jeff bikes to work').compute('root')
  doc.swap('Jeff', 'John') //use root-forms
  return doc.text()
}
// returns "John bikes to work" ✅
```

```
{
  let doc = nlp("Jeff's bikes are at work").compute('root')
  doc.swap('Jeff', 'John') //use root-forms
  return doc.text()
}
// returns "Jeff's bikes are at work" ❓ when I would hope for "John's bikes are at work" 
```

This can be isolated further:
```
{
  let doc = nlp("Jeff's").compute('root')
  doc.swap('Jeff', 'John') //use root-forms
  return doc.text()
}
// returns "Jeff's" ❓ when I would hope for "John's" 
```

The identical match behavior works like I'd expect
```
{
  let doc = nlp("Jeff's bikes are at work").compute('root')
  let m = doc.match('Jeff') //use root-forms
  return m.text()
}
// returns "Jeff's"
```

I've been beating my head up against the wall with this for a while so if there is even a workaround I would be THRILLED.  As of current, my workaround is doing a 
```
{
  let doc = nlp("Jeff bicycles to work.  Jeff's bikes are at work").compute('root')
  let m = doc.match('Jeff')
  doc.swap("Jeff", "John") // For the singular
  doc.swap(doc.possessives().match("Jeff").out("text"), basicPossessiveApostropheFormat("John")).  // For the possessive
  return doc.text()
}
// returns "John bicycles to work.  John's bikes are at work" ✅ but doesn't provide the Possessive tag to the new word.
```

My hacky solution loses the "Possessive" tag during the swap (i.e. Jeff has the "Possessive" tag, and John does not).

**Comment by spencermountain** [comment]
Hey Chad - yep this looks like a bug. I'm happy to fix it.
Will at out to the next release.
Cheers

**Comment by spencermountain** [comment]
fixed in 14.9.0
cheers
